### DISCUSSION TITLE
🖋️: 学習コンテンツかSantokuAppにページングの観点を入れられないか？

### DISCUSSION THREAD

**Discussion by tani-takanori** [ws-4020/mobile-app-crib-notes#234] [Ideas]
学習コンテンツでもFlatListを利用しており、スクロールでフェッチするパターンが実プロジェクトでも想定されるのでプロジェクト側に利用できる方式、もしくはノウハウを提供できないか。

ページングを入れる場合、今のように全量からFilterで分けるパターンのみだと「未完了タブ」がなかなか取得できないケースも考えられるので学習コンテンツに組み込むことも想定したい

**Comment by creature-water-valley** [comment]
アプリケーション方式やADRに以下のコンテンツを追加しているので、クローズ

- [ページネーション](https://ws-4020.github.io/mobile-app-crib-notes/react-native/santoku/application-architecture/http-api/http-api-tanstack-query#%E3%83%9A%E3%83%BC%E3%82%B8%E3%83%8D%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3%E3%81%A8%E7%84%A1%E9%99%90%E3%82%B9%E3%82%AF%E3%83%AD%E3%83%BC%E3%83%AB%E6%96%B9%E5%BC%8F)
- [ページネーションや無限スクロールへの対応](https://ws-4020.github.io/mobile-app-crib-notes/react-native/santoku/decisions/adr-013-react-query#%E3%83%9A%E3%83%BC%E3%82%B8%E3%83%8D%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3%E3%82%84%E7%84%A1%E9%99%90%E3%82%B9%E3%82%AF%E3%83%AD%E3%83%BC%E3%83%AB%E3%81%B8%E3%81%AE%E5%AF%BE%E5%BF%9C)
